app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,8 @@
   </head>
 
   <body>
-    <%= yield %>
+     <p class="notice"><%= notice %></p>
+     <p class="alert"><%= alert %></p>
+     <%= yield %>
   </body>
```

Comment:
- [ ] The doesn't run as expected, kindly fix routes so that first-time users will see the registration page instead of a blank page.

![devise](https://github.com/TaiwoEnoch/Social_Media_App/assets/53124548/2b6f376b-8104-4850-9d23-a4b896ed49d0)

Reply:
Hi @elisha2kyakpo1, I don't seem to understand what you are trying to bring up that my app is not running whereas is perfectly working on my machine, just finished running it now and it's working on my machine please can you be clear in your review, so I will know how to implement the changes?

<img src="
![Screen Shot 2023-07-08 at 08 00 20](https://github.com/TaiwoEnoch/Social_Media_App/assets/111912283/481f88e6-66ee-4de3-8f55-0e287fe6186a)
">

Reply:
@TaiwoEnoch, the purpose of adding devise is to handle user authentication such as sign in and sign up. A user without an account with your app and not logged in should see the page I have attached here(image below), but I only see this page after `typing http://localhost:3000/users/sign_in`. Kindly be reminded that an app should run on any machine regardless of the user, in this case, if you claim that your app runs well on your machine and fails on mine that should be a security issue to be worked on.

Reply:
@TaiwoEnoch, When I look at `app/views/layouts/application.html.erb`, it looks like it is missing some lines of code that come with Rails for the smooth running of a Rails app. The application.html.erb file is responsible for the HTML rendering in the browser.

Reply:
Noted @elisha2kyakpo1 
Thanks for your response